Makefile

```diff
@@ -52,16 +52,16 @@ all: generate fix vet fmt manifests test lint license misspell tidy bin/kube-app
 ## --------------------------------------
 
 $(TOOLBIN)/controller-gen: $(TOOLBIN)/kubectl
-	GOBIN=$(TOOLBIN) GO111MODULE=on go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.4.0
+	GOBIN=$(TOOLBIN) GO111MODULE=on go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.4.1
```

Comment:
This should be v0.5.0

https://github.com/kubernetes-sigs/controller-tools/releases/tag/v0.5.0